tests/conftest.py

```diff
@@ -235,11 +236,7 @@ def append_envelope(envelope):
 @pytest.fixture
 def capture_events_forksafe(monkeypatch, capture_events, request):
     def inner():
-        in_process_events = capture_events()
-
-        @request.addfinalizer
-        def _():
-            assert not in_process_events
+        capture_events()
```

Comment:
Need to remove this assertion because we actually do have events in the same process now.

We need to call capture_events or otherwise our transport will raise an error.